tests/unit/FirstStep.spec.js

```diff
@@ -0,0 +1,79 @@
+import { mount } from "@vue/test-utils";
+import FirstStep from "@/Components/FirstStep.vue";
+
+describe("FirstStep.vue", () => {
+    const wrapper = mount(FirstStep);
+    it("Has the main div tag", () => {
+        expect(wrapper.contains(".step-content")).toBe(true);
+    });
+
+    it("renders without any errors", () => {
+        expect(wrapper.isVueInstance()).toBeTruthy();
+    });
+
+    it("Checks if the cardtext function return the correct answer when selected is true", () => {
+        wrapper.setProps({
+            selected: true
+        });
+
+        expect(wrapper.vm.cardText).toBe("stepper.FS.selected");
+    });
+
+    it("Checks if the cardtext function return the correct answer when selected is false", () => {
+        wrapper.setProps({
+            selected: false
+        });
+
+        expect(wrapper.vm.cardText).toBe("stepper.FS.not-selected");
+    });
+
+    it("Checks if the yesText function returns correct answer", () => {
```

Comment:
This test and the next one have the same name, please come up with a different name that is more descriptive of each of them

Reply:
Thanks for the review @akmadian . I'll make the changes as conveyed.